scripts/unit-tests.sh

```diff
@@ -9,25 +9,34 @@
 # Disable colored output to avoid dirtying the log
 export GTEST_COLOR=no
 export SOFA_COLOR_TERMINAL=no
+export PATH_RESET="$PATH"
 
 usage() {
-    echo "Usage: unit-tests.sh (run|print-summary) <build-dir> <src-dir> [references-dir]"
+    echo "Usage: unit-tests.sh (run|print-summary) (unit|regression) <build-dir> <src-dir>"
 }
 
 if [ "$#" -ge 3 ]; then
     SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
     . "$SCRIPT_DIR"/utils.sh
 
     command="$1"
-    build_dir="$(cd $2 && pwd)"
-    src_dir="$(cd $3 && pwd)"
+    build_dir="$(cd $3 && pwd)"
+    src_dir="$(cd $4 && pwd)"
     if vm-is-windows; then
-        src_dir="$(cd $3 && pwd -W)"
+        src_dir="$(cd $4 && pwd -W)"
     fi
+    test_name_min="$2"
     test_type="unit-tests"
-    if [ -n "$4" ]; then
+    if [ "$2" == "regression" ]; then
         test_type="regression-tests"
-        references_dir="$4"
+        if vm-is-windows; then
+            # Avoid "libpython3X not found"
+            if [ -e "$VM_PYTHON3_EXECUTABLE" ]; then
+                pythonroot="$(dirname $VM_PYTHON3_EXECUTABLE)"
+                pythonroot="$(cd "$pythonroot" && pwd)"
+                export PATH="$pythonroot:$pythonroot/DLLs:$pythonroot/Lib:$PATH_RESET"
```

Comment:
```suggestion
                export PATH="$pythonroot:$pythonroot/DLLs:$pythonroot/Lib:$PATH"
```
why do you need to store in PATH_RESET?

Reply:
I reset the PATH at the end of the script. It is here not very useful given the fact that this is the last stage of the ci script, but at one point if in the future we add steps, it is better to clean up after...